libs/vscode-extension/src/lib/models/project.ts

```diff
@@ -78,6 +78,7 @@ export interface IProjectWizardContext extends IActionContext {
   targetFramework?: string | string[];
   isCustomCodeLogicApp?: boolean;
   logicAppName?: string;
+  skipWorkflowStateTypeStep?: boolean;
```

Comment:
NIT: Did it have you any kind of error if you didn't added to the IProjectWizardContext?

Reply:
No, I recently tested this and it seems to work without it